src/components/atoms/Dropdown/Dropdown.scss

```diff
@@ -37,12 +41,12 @@
 
     &__menu {
       background: var(--content--under-lighter-20pp);
-      box-shadow: 0 10px 20px opaque-black(0.25);
+      box-shadow: 0 10px 20px var(--content--under-25a);
```

Comment:
i think we can use a box-shadow function from `constants.scss` or add one there, but it's up to you

Reply:
Good idea